Backpack/NavigationBar/Classes/BPKNavigationBar.m

```diff
@@ -346,10 +354,38 @@ - (UIViewController *_Nullable)findNearestViewController {
     return (UIViewController *)responder;
 }
 
+- (void)updateAppearance {
+    self.backgroundView.backgroundColor = self.backgroundViewColor;
+    
+    self.leftButton.linkContentColor = self.contentColor;
+    self.rightButton.linkContentColor = self.contentColor;
+    self.titleView.titleLabel.textColor = self.contentColor;
+
+    if (self.largeTitleTextColor) {
+        self.largeTitleView.titleLabel.textColor = self.largeTitleTextColor;
+    } else {
+        self.largeTitleView.titleLabel.textColor = self.contentColor;
+    }
+}
+
 - (UIColor *)borderViewBackgroundColor {
     return [BPKColor dynamicColorWithLightVariant:BPKColor.surfaceHighlightColor darkVariant:BPKColor.textSecondaryColor];
 }
 
+- (UIColor *)contentColor {
```

Comment:
Is there a way to override the content color? (Same as title color).
(Should we want that?)

Reply:
Here I just want to provide a computed property for function `updateAppearance`, is this the right way to do it?